code/modules/hydroponics/gene_modder.dm

```diff
@@ -43,25 +43,31 @@
 
 /obj/machinery/plantgenes/RefreshParts() // Comments represent the max you can set per tier, respectively. seeds.dm [219] clamps these for us but we don't want to mislead the viewer.
 	for(var/obj/item/weapon/stock_parts/manipulator/M in component_parts)
-		if(M.rating > 3)
-			max_potency = 95
-		else
-			max_potency = initial(max_potency) + (M.rating**3) // 51,58,77,95 	 Clamps at 100
+		// 50,65,80,95
+		// Max natural potency is 100, machine clamps at 95
+		max_potency = min(initial(max_potency) + M.rating * 15, 95)
 
-		max_yield = initial(max_yield) + (M.rating*2) // 4,6,8,10 	Clamps at 10
+		// 4,6,8,10
+		// Max natural yield is 10
+		max_yield = min(initial(max_yield) + M.rating * 2, 10)
 
 	for(var/obj/item/weapon/stock_parts/scanning_module/SM in component_parts)
-		if(SM.rating > 3) //If you create t5 parts I'm a step ahead mwahahaha!
-			min_production = 1
-		else
-			min_production = 12 - (SM.rating * 3) //9,6,3,1. Requires if to avoid going below clamp [1]
+		// 7,5,3,1
+		// Min natural production is 1
+		min_production = max(initial(min_production) - SM.rating * 2, 1)
```

Comment:
Not sure about dropping base production time by 2

Reply:
Production and the growing cycle time is linear. A production value of 7 is still pretty insignificant, 9 is just completely useless, so this value doesn't matter much at Tier 1 parts.